File: app/src/main/java/com/work/lazxy/writeaway/WriteAway.java
package com.work.lazxy.writeaway;

import android.app.Application;
import android.content.Context;
import android.util.Log;

/**
 * Created by Lazxy on 2017/4/27.
 */

public class WriteAway extends Application{
    public static Context appContext;

    @Override
    public void onCreate(){
        super.onCreate();
        appContext = this;
    }
}

File: app/src/main/java/com/work/lazxy/writeaway/WriteAway.kt
package com.work.lazxy.writeaway

import android.app.Application
import android.content.Context

/**
 * Created by Lazxy on 2017/4/27.
 */
class WriteAway : Application() {
    override fun onCreate() {
        super.onCreate()
        appContext = this
    }

    companion object {
        lateinit var appContext: Context;
    }
}